cogs/utils/drawdota.py

```diff
@@ -721,6 +722,44 @@ def get_datetime_cell(match, region_data):
 		TextCell(str_time, font_size=18, horizontal_align="center")
 	)
 
+async def draw_meta_table(sorted_heroes, heroes): 
+	"""Takes a list of [heroes+p/b%+wr%] and draws a nice little discord-friendly table"""
+	
+	border_size = 10
+	grey_color = "#BBBBBB"
+	table = Table(background=discord_color2)
+	#Header
+	headers=[
+		TextCell(""), 
+		TextCell("Hero", padding=0), 
+		TextCell("Win %"), 
+		TextCell("Pick/Ban %")
+	]
+
+	table.add_row(headers)
+	for cell in table.rows[0]: 
+		cell.background = discord_color1
+
+	table.add_row([ColorCell(color=discord_color1, height=6) for i in range(len(headers))])
+	first = True
+
+	for hero in sorted_heroes: 
+		table.add_row([
+			ImageCell(img=await get_hero_image(hero["hero_id"]), height=48), 
+			TextCell(get_hero_name(hero["hero_id"]), font_size=24),
+			TextCell("%.2f" % get_hero_winrate(hero), fontsize=24),
+			TextCell("%.2f" % get_hero_pickban_percent(hero, heroes), fontsize=24)
```

Comment:
Same comment as above, you can use python string interpolation for this, i think the syntax for that is like f"{thingie:.2f}" or something

Reply:
Also! When showing percentages, I think people would prefer to see 50% rather than .50,